ocular_cli/src/commands/chains.rs

```diff
@@ -24,6 +26,7 @@ use self::show::ShowCmd;
 pub enum ChainsCmd {
     Show(ShowCmd),
     List(ListCmd),
+    Editor(EditorCmd),
```

Comment:
Ooh, that'll be cool. The reason it's editor is because there was supposed to be an edit command where the user can edit the `toml` file without the need to open an editor. However, since we're noting continuing with the edit command it'll be nice to change this to edit.

Reply:
done!